### PR TITLE
Add networking subtabs and refactor out unnecessary loadLinode calls

### LINKED ISSUE #185: Implement Linode networking tabs

### PR DESCRIPTION
**Pull request by eatonphil**
Closes #185

screen shot 2016-11-02 at 3 29 41 pm

### CODE REVIEW

**Comment by abemassry** [comment]
include new location of NetworkingPage.spec.js

**Comment by eatonphil** [comment]
screen shot 2016-11-03 at 2 34 55 pm

File: src/linodes/linode/layouts/IndexPage.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ import { setSource } from '~/actions/source';
 
 export function getLinode() {
   const { linodes } = this.props.linodes;
-  const linodeId = parseInt(this.props.params.linodeId);
+  const linodeId = parseInt(this.props.params.linodeId, 10);
```
Comment (abemassry) [inline src/linodes/linode/layouts/IndexPage.js]: Is this for a max length?

Comment (eatonphil) [inline src/linodes/linode/layouts/IndexPage.js]: no, it's a semi-optional field indicating the number base

**Comment by eatonphil** [comment]
thanks!

File: src/linodes/linode/layouts/IndexPage.js (hunk 1 above)
Comment (briansteffens) [inline src/linodes/linode/layouts/IndexPage.js]: Let's make all our IDs pentadecimal and use emojis for the digits.

Comment (eatonphil) [inline src/linodes/linode/layouts/IndexPage.js]: 😂

**Pull request merged**